Security/src/ExchangeExtendedProtectionManagement/ExchangeExtendedProtectionManagement.ps1

```diff
@@ -276,9 +277,20 @@ begin {
             Show-Disclaimer @params
         }
 
-        Write-Verbose ("Running Get-ExchangeServer to get list of all exchange servers")
-        Set-ADServerSettings -ViewEntireForest $true
```

Comment:
@dpaulson45 is there a reason we used the cmdlet here?

Reply:
Yep, `Get-Mailbox`, `Get-Recipient` etc. . I will leave it removed and will not add it again.